deploy/crds/performance.openshift.io_performanceprofiles_crd.yaml

```diff
@@ -34,15 +34,22 @@ spec:
             cpu:
               description: CPU defines set of CPU related parameters.
               properties:
+                balanceIsolated:
+                  description: BalanceIsolated toggles whether or not the Isolated
+                    CPU set is eligible for load balancing work loads. When this option
+                    is set to "false", the Isolated CPU set will be static, meaning
+                    workloads have to explicitly assign each thread to a specific
+                    cpu in order to work across multiple CPUs. Setting this to "true"
+                    allows workloads to be balanced across CPUs. Setting this to "false"
+                    offers the most predictable performance for guaranteed workloads,
+                    but it offloads the complexity of cpu load balancing to the application.
+                    Defaults to "true"
```

Comment:
Should it default to false, to preserve the previous behavior?

Reply:
Nobody uses it yet and the previous behavior was actually one of the issues. I would prefer enabled balancing by default myself.